apps/css-workshop/pages/toggle-switch.html

```diff
@@ -196,6 +225,10 @@ <h2>Small</h2>
               role="switch"
               disabled
             />
+            <svg-checkmark
+              class="iui-toggle-switch-icon"
+              aria-hidden="true"
+            ></svg-checkmark>
```

Comment:
Was this checkmark intentionally not added to all toggle switches on the page? E.g. "Toggle switch on left".